### PR TITLE
create start up guide section for macos and linux

### PR DESCRIPTION
**Pull request by tesinormed**
very good

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,14 @@ It is set up to get and store a day-by-day count of views and clones that does n
 I'm not a superhero that can retrieve data that is more than 2 weeks old unless it has already been stored by the program, then it merely keeps going from the first time logging started.  
 
 ## Required Steps:  
-go into ViewsGetter.bat and in the quotes, add a username, and the password (see note at end)  
-in cmd, do "javac *.java" in the directory that the .java files are in to compile the java files
```
Comment (jojo2357) [inline README.md:8]: left out the compile ur java stuff

**Review by jojo2357 — CHANGES_REQUESTED** [review]
instructions left out compiling java stuff

File: README.md (hunk 1 above)
Comment (tesinormed) [reply README.md:8]: should i make the batch script compile for you or should i add those instructions back

Comment (jojo2357) [reply README.md:8]: i thought for batch you just clicked it and it ran

Comment (jojo2357) [reply README.md:8]: oh i see what you mean sorry. Id rather not cuz you only need to compile when u download everything in theory

Comment (tesinormed) [reply README.md:8]: hmm

Comment (tesinormed) [reply README.md:8]: ~~but what if we can remove that step and make it as simple and straightforward as we can~~

Comment (jojo2357) [reply README.md:8]: alright lets do that. Merge time!

Comment (tesinormed) [reply README.md:8]: bruh now i gotta make another pr that adds that compiling option

**Pull request merged**